Kickstarter-iOS/Views/Controllers/DiscoveryFiltersViewController.swift

```diff
@@ -60,6 +60,7 @@ internal final class DiscoveryFiltersViewController: UIViewController, UITableVi
       .observeValues { [weak self] isVisible in
         if isVisible {
           self?.dataSource.loadCategoriesLoaderRow()
+          self?.filtersTableView.reloadData()
```

Comment:
Agree, I've changed to to hide on `values()`. Tested it on my device and behaviour seems the same with/without the reloading!